docs/index.md

```diff
@@ -1300,6 +1301,45 @@ class ExplicitFilePathSettings(BaseSettings):
         )
 ```
 
+### Azure Key Vault
+
+You must set two parameters:
```

Comment:
we need to be clear about the limitations relative to other sources, if we can't get all KV pairs easily.

Reply:
@AndreuCodina if we can find a way to fetch all the key/value first and build a dict, we can easily use the logic that we have in `EnvSettingsSource`. otherwise, we have to be clear and document what exactly we support and what we not support relative to other sources.

Reply:
Done.